README.md

```diff
@@ -14,8 +14,6 @@ The goal of this repo and plugin is to provide users with the BBCode suite that
 
 🚧 = In progress or needs CSS.
 
-✔️ = Complete.
```

Comment:
Removed the checkmark as it was not always committed in README updates and unnecessary.